hail/python/hailtop/batch/batch_pool_executor.py

```diff
@@ -232,7 +232,7 @@ async def async_map(
     ) -> AsyncGenerator[int, None]:
         """Aysncio compatible version of :meth:`.map`."""
         if not iterables:
-            return (x for x in range(0))
+            return the_empty_async_generator()
```

Comment:
the lint didn't like that I used an empty generator where an async generator was expected.